blueman/gui/manager/ManagerDeviceMenu.py

```diff
@@ -213,6 +252,32 @@ def Generate(self):
             self.append(item)
             return
 
+        # Generic (dis)connect
+        # LE devices do not appear to expose certain properties like uuids until connect to at least once.
+        # show generic connect for these as we will not show any way to connect otherwise.
+        device_uuids = self.SelectedDevice['UUIDs']
+        show_generic_connect = not device_uuids
+        for uuid in device_uuids:
+            if ServiceUUID(uuid).short_uuid in (
+                AUDIO_SOURCE_SVCLASS_ID, AUDIO_SINK_SVCLASS_ID, GENERIC_ACCESS_SVCLASS_ID,
+                HANDSFREE_AGW_SVCLASS_ID, HEADSET_SVCLASS_ID, HID_SVCLASS_ID
```

Comment:
Ok, Added a new commit that adds the missing audi gateway, hog and midi services. Will squash with the main commit before merging.

My reasoning for adding midi is that we never provided this service before as we had no knowledge of the proprietary uuid. If the users runs a recent enough bluez version connect will succeed if not no loss compared to what we would have provided before.

The midi service check is imho a bit ugly but I am not sure how else to best handle this.

Awaiting further investigation that adds more work for me 😝.